sample/src/main/java/kin/sdk/core/sample/Utils.java

```diff
@@ -18,4 +21,25 @@ public static void copyToClipboard(Context context, CharSequence textToCopy) {
             clipboard.setPrimaryClip(clip);
         }
     }
+
+    public static int getScreenHeight(Activity activity) {
+        Display display = activity.getWindowManager().getDefaultDisplay();
+        Point size = new Point();
+        display.getSize(size);
+        int height = size.y;
+        return height;
```

Comment:
prefer a single getScreenSize method that returns Point instead of code duplication.

Reply:
that class the use it doenst need to know about the format of the answer
for now we only need the height so i can remove the the getScreen width if you prefer

Reply:
remove